CombinatorialGames/Game/Special.lean

```diff
@@ -130,4 +130,27 @@ theorem star_fuzzy_down : star ‖ down := by
   rw [← neg_fuzzy_neg_iff, neg_down, neg_star]
   exact star_fuzzy_up
 
+section TinyMiny
```

Comment:
Do we need a `section` for this material? (conversely, would it be useful to `section` off the other special games?)

Also, I believe the convention is to put the section docstring before the `section`.

Reply:
I added section to isolate the variable declaration: it might be good to add section to all the special game categories.

Reply:
I'm generally not a fan of explicit `variable`s. They're a bit easy to forget and as the API develops you usually end up with theorems which shouldn't have explicit variables but do. But that's a minor style complaint.